src/DelegateListOverview/DelegateListOverview.tsx

```diff
@@ -0,0 +1,378 @@
+        CurrentPage: 1,
+        HasNext: undefined,
+        HasPrevious: undefined,
+        PageSize: undefined,
+        TotalPages: 1,
+        TotalCount: undefined
+    });
+
+    // Standard API GET Parameters
+    const [requestDelegateOffersOptions, setRequestDelegateOffersOptions] = React.useState<RequestDelegateOffersOptions>({
+        MaxAvailableForInMonth: undefined,
+        MaxPrice: undefined,
+        MinAvailableForInMonth: undefined,
+        MinPrice: undefined,
+        PageNumber: 1,
+        PageSize: 9,
+        RegionQuery: "",
+        SearchQuery: ""
+    });
+
+    // useEffect is only being used on page load. Here it loads the first page of the api call with no filters applied.
+    useEffect(() => {
+        handleLoadDelegates();
+    }, []);
+
+    // Single function to handle the API call with the data and headers.
+    function handleLoadDelegates() {
+        setDelegateCards(<div><Spinner animation="border" variant="primary"/></div>);
+
+        loadDelegates(requestDelegateOffersOptions).then(r => {
+            mapJsonToTSX(r.data);
+            let customHeaders = r.headers;
+            setHeaders(customHeaders);
+            mapPagesToTSX(customHeaders);
+        })
+    }
+
+    // The actual URL Builder and API Call.
+    const loadDelegates = async (reqOptions: RequestDelegateOffersOptions) => {
+        // Request headers and other parameters
+        const requestOptions: any = {
+            method: 'GET',
+            headers: {'Content-Type': 'application/json'},
+            mode: "cors",
+            cache: "default",
+        };
+
+        // Bare minimum URL
+        let url = `${config.SERVICES.DELEGATE}?PageNumber=${reqOptions?.PageNumber.toString()}&PageSize=${reqOptions?.PageSize}`
+
+        // Additional components based on if they are populated or not
+        if (reqOptions.MaxAvailableForInMonth) url += `&MaxAvailableForInMonth=${reqOptions.MaxAvailableForInMonth}`
+        if (reqOptions.MinAvailableForInMonth) url += `&MinAvailableForInMonth=${reqOptions.MinAvailableForInMonth}`
+        if (reqOptions.MaxPrice) url += `&MaxPrice=${reqOptions.MaxPrice}`
+        if (reqOptions.MinPrice) url += `&MinPrice=${reqOptions.MinPrice}`
+        if (reqOptions.SearchQuery !== "") url += `&searchQuery=${reqOptions.SearchQuery}`
+        if (reqOptions.RegionQuery !== "") url += `&regionQuery=${reqOptions.RegionQuery}`
+
+        // The API Call
+        let response = await fetch(url, requestOptions)
+        let data = await response.json();
+
+        return {headers: JSON.parse(response["headers"].get('X-Pagination') as string), data}
+    }
+
+    // The mapping of delegate objects from the json to HTML code
+    const mapJsonToTSX = (json: DelegateOffer[]) => {
+        let mappedItems = json.map((item, key) => {
+            return <div className="col-lg-4 d-flex align-items-stretch" style={{marginTop: '30px'}} key={item.id}>
```

Comment:
Waarom gebruik je niet <Col lg={4} van import "react-boostrap".

Styling ook het liefst in een apart css file.